tasks/styles.js

```diff
@@ -55,6 +57,9 @@ module.exports = (gulp, $, pkg) => {
       .pipe($.if(options.production, $.rename({ suffix: '.min' })))
       .pipe($.if(options.concat, $.concat(pkg.title.toLowerCase().replace(/[^a-z]/g,'') + '.css')))
       .pipe($.cssUrlCustomHash({
+        customHash: (fileName, hash, filePath) => {
+          return path.basename($.twigAsset.asset(filePath));
+        },
```

Comment:
This code doesn't change the parameter name from `_` to `v` yet?  Is that the remaining todo?

Reply:
🙈 seems I missed it… thanks

Reply:
Ah, now I see. The query string is returned by the `$.twigAsset.asset()` function